README.md

```diff
@@ -71,11 +71,28 @@ The returned event payload is the same as the input.
 `SendRichMessageFailed`
 ```json
 {
-  "inputMessage": { ... },
+  "inputMessage": {
+  },
   "error": "..."
 }
 ```
 
+
+### GetReactionMessageInfo
```

Comment:
Trying to understand use case for this one and what it actually does 🤔

Reply:
problem with the previous implementation: JSON standard does not allow such tokens. 
To fix this, I have removed the ... token. 
> Trying to understand use case for this one and what it actually does 🤔